src/ER/ER.scss

```diff
@@ -0,0 +1,40 @@
+@import '~uswds/dist/scss/packages/required';
+@import '../../theme/_uswds-theme-color';
+@import '../../theme/_uswds-theme-components';
+
+.usa-card__container { 
+    width: 500px;
+}
+
+.usa-card__header > :only-child, .usa-card__body > :only-child {
+    float: left; 
+}
+
+.usa-card__body > :only-child { 
+    font-size: 20px;
+}
+
+.header__time {
```

Comment:
Redefining header__time here will override the styles needed for it in the Header component. Causing the clock in the header to be the wrong color and position.

The convention I've been using is to define _shared_ styles in the theme folder in the custom-styles css file, since this will be loaded in the page first.

Then, any tweaks on shared styles in individual component stylesheet files should use unique names related to their component. Fore example, instead of putting this in the ER stylesheet, create a unique classname with BEM convention (i.e. cardbody__time) and create a CardBody.scss file that customizes this element for the CardBody component (where it is actually used).

That way, it should always be easy to know where/what is styling something. Otherwise, what happened was that I noticed that the header clock styling was now broken, and I had to inspect in the browser and trace back through the files to find out what was going on.

Does that make sense?